keyboards/tweetydabird/lotus58/blok/rules.mk

```diff
@@ -0,0 +1,2 @@
+BOOTLOADER = caterina
```

Comment:
```suggestion
```
The converter implementation will override `BOOTLOADER` to the appropriate value anyway.